respect-app-compose/src/androidMain/AndroidManifest.xml

```diff
@@ -33,7 +34,17 @@
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
+            <intent-filter android:autoVerify="true">
+                <action android:name="android.intent.action.VIEW" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <data android:scheme="https" />
+                <data android:host="testproxy.devserver3.ustadmobile.com" />
```

Comment:
Needs to add *. host

Reply:
Needs to be added, not replaced.
eg. should have testproxy.devserver and *.testproxy.devserver